### DISCUSSION TITLE
TOC on Blog

### DISCUSSION THREAD

**Discussion by Sidon** [thuliteio/doks#953] [Q&A] [answered]
After installing this project and creating a post on a blog, I tried to activate TOC with `toc: true`, and it's not worked. Is there an easy way to activate the TOC on the blog?

**Comment by h-enk** [comment, marked as answer]
You could add it (not tested) by adding https://github.com/h-enk/doks/blob/master/layouts/partials/sidebar/docs-toc.html#L3-L16 before https://github.com/h-enk/doks/blob/master/layouts/blog/single.html#L17

**Reply by Sidon** [reply]
Ok, it works, but after your tip, I decided to try the single layout of docs on the blog, and, at first, seems that it works better for my necessities, maybe with some adaptations. Thanks a lot!
